### ISSUE TITLE
warning: URI.escape is obsolete

### ISSUE THREAD

**Issue by sanak** [gtt-project/redmine_gtt_print#18]
**Problem**
When running a test on Ruby 2.7 environment, the following warning happen.
```
redmine_gtt_print/lib/redmine_gtt_print/mapfish.rb:xx: warning: URI.escape is obsolete
```

**To Reproduce**
Execute test as follows with launching mapfish print server locally:
```bash
bundle exec rake redmine:plugins:test RAILS_ENV=test NAME=redmine_gtt_print
```

**Expectation**
Those warnings should not happen.

**Comment by sanak** [comment]
From the  following Japanese article:
- [【翻訳】URI.escapeは非推奨メソッドです。あなたのクエリ文字列をパーセントエンコードするには](https://qiita.com/jnchito/items/dc9285884b2349833fad)

Just replacing `URI.escape` => `URI.encode_www_form_component`  may be enough, because there seems to be no case which over URL components (`/`).
```rb
    def get_capabilities(print_config)
      return unless print_config
      str = URI.escape(print_config)
      begin
        url = "#{@host}/print/#{str}/capabilities.json"
```
```rb
    def request_print(json, layout, format, referer = nil, user_agent = nil)
      str = URI.escape(layout)
      url = "#{@host}/print/#{str}/report.#{format}"
      if Rails.env.development?
```

**Comment by sanak** [comment]
I fixed this issue by PR #19, so I close this.
